docs/modules/devicetwin.md

```diff
@@ -0,0 +1,328 @@
+
+# DeviceTwin
+
+
+## Overview
+
+DeviceTwin is responsible for storing device status and syncing device status to the cloud. 
+It also provides query interfaces for applications.
+   
+   
+## Operations Performed By Device Twin Controller
+ 
+ The following are the functions performed by device twin controller :-
+  
+  - Sync Sql Lite
+  - Register and Start Modules
```

Comment:
before this , in the overview we can mention about the sub-modules and here we can say "Register and start sub-modules"